netmanager/src/views/pages/Settings/components/GenerateToken/GenerateToken.js

```diff
@@ -1,133 +1,432 @@
 import React, { useState, useEffect } from 'react';
+import Alert from '@material-ui/lab/Alert';
+import { CircularLoader } from 'views/components/Loader/CircularLoader';
+import PropTypes from 'prop-types';
+import FileCopyIcon from '@material-ui/icons/FileCopy';
+import DeleteIcon from '@material-ui/icons/DeleteOutlineOutlined';
+import usersStateConnector from 'views/stateConnectors/usersStateConnector';
+import { makeStyles } from '@material-ui/core/styles';
+import { createClientApi, getClientsApi, generateTokenApi } from '../../../../apis/analytics';
+import { useDispatch } from 'react-redux';
+import { updateMainAlert } from 'redux/MainAlert/operations';
+import DataTable from './Table';
 import {
   Card,
   CardHeader,
-  CardContent,
-  CardActions,
   Divider,
+  CardActions,
   Button,
+  Dialog,
+  DialogTitle,
+  DialogContent,
   TextField,
-  InputAdornment,
-  IconButton
+  DialogActions,
+  CircularProgress
 } from '@material-ui/core';
-import Alert from '@material-ui/lab/Alert';
-import { CircularLoader } from 'views/components/Loader/CircularLoader';
-import PropTypes from 'prop-types';
-import FileCopyIcon from '@material-ui/icons/FileCopy';
-import CloseIcon from '@material-ui/icons/Close';
-import usersStateConnector from 'views/stateConnectors/usersStateConnector';
-import { generateAccessTokenForUserApi } from '../../../../apis/accessControl';
+import CheckIcon from '@material-ui/icons/Check';
+import { getUserDetails } from '../../../../../redux/Join/actions';
```

Comment:
@OchiengPaul442 
Can we try to use some tools that take care of relative paths? I am worried that we can easily make mistakes with these long relative paths? 

Example of such tools (approaches) include but not limited to:

1. Using Babel and webpack: If you are using Babel and webpack to build your React application, you can configure aliases in your webpack configuration file. This allows you to define custom shortcuts for importing modules.
2. Using Create React App with CRACO: If you are using Create React App (CRA), you can use CRACO (Create React App Configuration Override) to customize the configuration without ejecting. CRACO allows you to modify the webpack configuration and add support for import aliases.
3. etc

Please note that these solutions require additional configuration and may depend on the specific build tools you are using. It's important to refer to the documentation of your build tools for more detailed instructions and examples.